docs/src/openapi-v1.yaml

```diff
@@ -1045,7 +1045,10 @@ paths:
         Get the status of the current auction.
       description: |
         Returns an overview of the current of auction. There is only one auction
-        at a time. If there is no auction most fields will be `null`.
+        at a time. If there is no auction most fields will be `null`. If the current
+        auction phase is in `vrfDelay` and you are looking to retrieve the winners,
```

Comment:
Just to check based on our chat; I guess you can retrieve the highest bidders, but perhaps you don't know the actual "winners" until after the vrfDelay? (I need to read up on auctions more!)

Reply:
So yea the actual "winners" are reflected by slots that are leased out. When `manage_auctions_end` is called in the auctions pallet, it [calculates the finals winners](https://github.com/paritytech/polkadot/blob/master/runtime/common/src/auctions.rs#L568) then it will [lease out the slots](https://github.com/paritytech/polkadot/blob/master/runtime/common/src/auctions.rs#L577). 

So Options for finding the current winners: Use the `AuctionClosed` event as an block indicator to query `/leases/current`, 

Query the last block before the `finishEnd` block in order to get the latest winning bids.

Reply:
Going to update it to "latest winning bids" to be more accurate

Reply:
fixed